routes/dbapi.js

```diff
@@ -50,9 +51,9 @@ db.prototype.readCollection = function(collection, callback) {
   });
 };
 
-db.prototype.handleInsertionError = function(err) {
-	console.log(err);
-	return 'Error';
+db.prototype.handleInsertionError = function(err, returnValue = 'Error') {
+  console.log(err);
```

Comment:
remove console, this often crashes the node server in prod or keep it conditional if NODE_ENV is development